wandb/apis/public.py

```diff
@@ -2237,6 +2237,45 @@ def entity(self):
 
     @property
     def state(self):
+        item = self.get_item()
+        if item:
+            return item["state"].lower()
+
+        raise ValueError(
+            f"Could not find QueuedRunItem associated with id: {self.id} on queue id {self.queue_id} at itemId: {self.id}"
+        )
+
+    @normalize_exceptions
+    def get_item(self):
+        query = gql(
```

Comment:
nit: Can we split the two query attempts into separate methods? I think that will make the `get_item` flow more clear